src/test/resources/features/03-login/login-test.feature

```diff
@@ -0,0 +1,11 @@
+Feature: User Login check
+  I want to be able to see if a valid user login correctly
+
+  Scenario: Open the home page fill in credentials and login
+    Given User opens the website at "https://openmrs-spa.org/openmrs/spa"
+    When User enters username "admin"
+      And User clicks "continue"
+      And User enters Password "Admin123"'
```

Comment:
hi @joachim-junior At the end of the line, I think there no purpose for `single inverted coma`

Reply:
Thank you